C++/2. Variables.md

```diff
@@ -40,25 +44,27 @@ You declare a variable by typing the variable type before the name of the variab
 int main() {
   bool boolean = true;
   char character = 'k';
-  short int smallNumber = 2;
   int number = -4;
-  unsigned int positiveNumber = 4;
-  long int largeNumber = 7000000;
   float decimal = 3.5;
-  long long int largerNumber = 70000000;
-  double bigDecimal = 3.456;
+  double big_decimal = 3.456;
+  short int small_number = 2;
+  unsigned int positive_number = 4;
+  long int large_number = 7000000;
+  long long int larger_number = 70000000;
 
-  std::cout << std::boolalpha // ignore boolalpha, it will be mentioned later
-      << number << std::endl
+  // ignore boolalpha, it will be mentioned later
+  std::cout << std::boolalpha;
```

Comment:
does this work? If so okay; if not just revert it i don't care you don't seem to understand because of my bad explaining skills..

That is all; I am going to merge this soon. Just do the finishing touches and I am done.

Reply:
Please, I don't care if it takes a week, take the time to adequately explain yourself.

Also yes, it works.

Reply:
If it works then nothing...